src/Components/HomePage.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { FC, useState } from 'react'
+import {
+    Box,
+    List,
+    Snackbar
+} from '@material-ui/core';
+
+import MuiAlert from '@material-ui/core/Alert';
+
+import Task from './Task';
+import TaskForm, { ITask } from './TaskForm';
+
+const HomePage: FC = () => {
+
```

Comment:
I suggest removing this line.

Reply:
Resolved on 5baffc6f194600ac5b866628ee4218f04877c7ac